tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -2772,6 +2863,46 @@ def test_const():
     #         return gv
 
 
+def test_constant_of_shape():
+    """Test case for constant_of_shape op."""
+    shape = [32, 16]
+    const_node = helper.make_node(
+        "ConstantOfShape",
+        ["shape"],
+        ["output"],
+        value=helper.make_tensor("value", TensorProto.FLOAT, [1], [1.0]),
+    )
+    graph = helper.make_graph(
+        [const_node],
+        "const_test",
+        inputs=[],
+        outputs=[helper.make_tensor_value_info("output", TensorProto.FLOAT, shape)],
+        initializer=[
+            helper.make_tensor("shape", TensorProto.INT64, [2], shape),
+        ],
+    )
+
+    model = helper.make_model(graph, producer_name="const_test")
+    mod = relax.from_onnx(model)
+
+    # pylint: off
+    # TODO: figure out how to get metadata to work with Relax TVMScript
```

Comment:
For metadata, you can print it out by using `mod.show(show_meta=True)`. I usually use a relax const in this case like `R.const(2, dtype="int64")`.